.travis.yml

```diff
@@ -6,8 +6,8 @@ services:
   - docker
 
 before_install:
-  - docker build -t duncdrum/exist-docker .
-  - docker run -it -d --name exist -p 8080:8080 duncdrum/exist-docker
+  - docker build -t existdb/existdb .
+  - docker run -it -d --name exist -p 8080:8080 existdb/existdb
 
 before_script:
   - sleep 35
```

Comment:
Out of interest, why do we need to sleep for *n* seconds. Seems rather indeterministic?

Reply:
healthcheck takes 30s to start, this way we can see the results on travis

Reply:
Sorry, but can you tell me, what is healthcheck?

Reply:
It reports the health status of your running container [docs](https://docs.docker.com/engine/reference/builder/#healthcheck)

Reply:
Fair enough :-)